### PR TITLE
[JTC] Add pid gain structure to documentation

### PR DESCRIPTION
**Pull request by christophfroehlich**
I also described the used control law briefly.

### CODE REVIEW

File: joint_trajectory_controller/doc/userdoc.rst [hunk 1]
```diff
@@ -140,29 +140,64 @@ open_loop_control (boolean):
 
   If this flag is set, the controller tries to read the values from the command interfaces on starting. If they have real numeric values, those will be used instead of state interfaces. Therefore it is important set command interfaces to NaN (std::numeric_limits<double>::quiet_NaN()) or state values when the hardware is started.
 
-constraints (structure)
+constraints (structure):
```
Comment (destogl) [inline joint_trajectory_controller/doc/userdoc.rst:143]: I am not sure what I like more, with or without `:`

You can decide :D

**Review by destogl — APPROVED** [review]
Thanks!

File: joint_trajectory_controller/doc/userdoc.rst (hunk 1 above)
Comment (christophfroehlich) [reply joint_trajectory_controller/doc/userdoc.rst]: you are right, looks nicer without ;)

**Review by bmagyar — APPROVED** [review]

**Comment by christophfroehlich** [comment]
@Mergifyio backport humble

**Comment by bmagyar** [comment]
@Mergifyio backport humble

**Pull request merged**